ruby/vampires.rb

```diff
@@ -0,0 +1,37 @@
+
+puts "How many employees are being processed?"
+number_of_employees_to_process = gets.chomp
+until number_of_employees_to_process == 0
+	puts "What is your name?"
+	employee_name = gets.chomp
+	puts "How old are you?" 
+	employee_age = gets.chomp
+	puts "What year were you born?"
+	employee_birth_year = gets.chomp
+	puts "Our company cafeteria serves garlic bread. Should we order some for you?"
+	want_garlic_bread = gets.chomp
+	puts "Would you like to enroll in the company health insurance?"
+	want_health_insurance = gets.chomp
+	allergy = nil
+	result = "Results inconclusive."
+	until (allergy == "done") || (allergy == "sunshine")
+		puts "Name any allergies one at a time please. If there are no more to list, please reply done."
+		allergy = gets.chomp
+		if allergy == "sunshine"
+			result = "Probably a vampire."
+	end
+
+	if ((employee_age.to_i == Date.today.year - employee_birth_year.to_i) || (employee_age.to_i - 1 == Date.today.year - employee_birth_year.to_i)) && ((want_garlic_bread=="yes") || (want_health_insurance=="yes"))
```

Comment:
I'd suggest removing any logic outside of your conditional statements so you can just evaluate a variable. It's common practice to evaluate booleans if your code allows it. We can take all of your logic out and turn the answers from the employee into booleans, for instance a long form would be: 
```
if garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
    garlic_bread = true
else
   garlic_bread = false
end
```
can be refactored to:
```
garlic_bread = garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
```
If `garlic_bread.downcase` is neither of those answers, `garlic_bread` will be set to false.